### PR TITLE
`portmap` support masquerade all 

### PR DESCRIPTION
**Pull request by BSWANG**
to #771

### CODE REVIEW

**Comment by dcbw** [comment]
@BSWANG we had some questions about the architecture in issue #771 if you have a chance to respond... Thanks!

File: plugins/meta/portmap/portmap.go [hunk 1]
```diff
@@ -259,13 +259,18 @@ func fillDnatRules(c *chain, config *PortMapConf, containerNet net.IPNet) {
 			hpRule := make([]string, len(ruleBase), len(ruleBase)+4)
 			copy(hpRule, ruleBase)
 
+			masqCIDR := containerNet.String()
+			if config.MasqAll {
+				masqCIDR = "0.0.0.0/0"
```
Comment (squeed) [inline plugins/meta/portmap/portmap.go:264]: Need to check `isV6` here too and generate the right CIDR

**Review by squeed — APPROVED** [review]

**Comment by squeed** [comment]
@BSWANG can you file a PR against https://github.com/containernetworking/cni.dev to document this please?

**Comment by MikeZappa87** [comment]
@BSWANG can you submit a PR to update the Docs?

**Comment by BSWANG** [comment]
> @BSWANG can you file a PR against https://github.com/containernetworking/cni.dev to document this please?

`cni.dev` PR https://github.com/containernetworking/cni.dev/pull/110

**Pull request merged**